e2e-version/index.js

```diff
@@ -73,6 +72,7 @@ async function run() {
 
     console.log('Resolved images: ', images);
     core.setOutput(MatrixOutput, JSON.stringify(images));
+    return images;
```

Comment:
Returning these doesn't change things when running the Action in GH, but it will make unit testing a lot easier.

Reply:
Alternatively you mock the `core` dependency and inspect it. But I'm fine with either way as long as we have tests that verify the behaviour :)